File: consumer/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/consumer/exceptions/SikkerhetsBegrensningException.java
package no.nav.sbl.dialogarena.soknadinnsending.consumer.exceptions;

import no.nav.sosialhjelp.soknad.domain.model.exception.SosialhjelpSoknadApiException;

public class SikkerhetsBegrensningException extends SosialhjelpSoknadApiException {
    public SikkerhetsBegrensningException(String message, Exception exception) {
        super(message, exception);
    }
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/client/exceptions/SikkerhetsBegrensningException.kt
package no.nav.sosialhjelp.soknad.client.exceptions

import no.nav.sosialhjelp.soknad.common.exceptions.SosialhjelpSoknadApiException

class SikkerhetsBegrensningException(message: String?, exception: Exception?) :
    SosialhjelpSoknadApiException(message, exception)
